src/elements/content-preview/ContentPreview.js

```diff
@@ -1145,15 +1145,18 @@ class ContentPreview extends React.PureComponent<Props, State> {
         });
     };
 
-    handleAnnotationSelect = ({
-        file_version: { id: annotationFileVersionId },
-        target: { location = {} },
-    }: Annotation) => {
+    handleAnnotationSelect = (annotation: Annotation) => {
+        const {
```

Comment:
Did this need to move out of the function signature?

Reply:
It seemed clearer to move it out and pull the data from `annotation` and then emit the whole annotation to `content-preview`